cypress/integration/createTrip.spec.js

```diff
@@ -0,0 +1,29 @@
+describe('Create a trip', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+    cy.get('a > button').click()
+    cy.get('form > input[name="title"]').type('Barcelona')
+    cy.get('form > input[name="date"]').type('01.05. - 20.05.2020')
+    cy.get('form > button').click()
+  })
+
+  it('visite homepage after creation', () => {
+    cy.url().should('eq', 'http://localhost:3000/')
+  })
+
+  it('the new card is visible after creation', () => {
+    cy.visit('http://localhost:3000/')
+    cy.get('section > a > section > div > h2').contains('Barcelona')
+    cy.get('section > a > section > div > div > span').contains(
+      '01.05. - 20.05.2020'
+    )
+  })
+
+  it('the new card is visible after reload', () => {
+    cy.visit('http://localhost:3000/')
+    cy.get('section > a > section > div > h2').contains('Barcelona')
+    cy.get('section > a > section > div > div > span').contains(
+      '01.05. - 20.05.2020'
+    )
+  })
```

Comment:
Maybe try to write the it descriptions in a way that reads like a sentence? For example
- '(it) is redirected to the homepage after creation' 
- '(it) shows the new card'
- '(it) shows the card after reload'